data/indonesia/bbc_indonesian_radio/liveradio.json

```diff
@@ -1,7 +1,12 @@
 {
     "metadata": {
         "id": "urn:bbc:ares::asset:indonesia/bbc_indonesian_radio/liveradio",
-        "tags": {}
+        "tags": {},
+        "language": "id-ID",
+        "analyticsLabels": {
+            "pageIdentifier": "indonesia.bbc_indonesian_radio.liveradio.page",
```

Comment:
`pageIndentifier` seems to map to `counterName` in the other types of Ares payloads (Indexes and CPS pages) is there a reason we wouldn't want this to remain consistent?

Reply:
I believe it's called `counterName` in front page, but `pageIdentifier` in articles

https://github.com/bbc/simorgh/blob/82b171addc7c4085ff706fa9483367a1a3eb791d/src/app/containers/ATIAnalytics/params/article/index.jsx#L28

Even though frontpage does call it `counterName`, it gets remapped to `pageIdentifier` before being used in analytics

https://github.com/bbc/simorgh/blob/2c129c231039a63a8c7d5576ca182b8f9133534e/src/app/lib/analyticsUtils/frontpage/index.js#L3

So I think frontpage is the thing that is inconsistent here

The actual data source is temporary in this PR anyway (which is why I only did it for one service) - we acknowledged once the ARES work is complete, there will be some integration updates required (analytics, morph module, `getInitialData`, other places)

Reply:
I believe `pageIdentifier` is the human readable version in the code but Ares will return it as `counterName` for real data so ideally we should have the fixture payloads we use for liveradio match the Ares response for CPS MAPs and on-demand radio

Reply:
Fixture data is a different (blocked) ticket: https://github.com/bbc/simorgh/issues/2754

I can block this PR too behind ARES if preferred though - have asked Iain to provide a sample in the mean time